packages/agent/src/hd-identity-vault.ts

```diff
@@ -498,28 +508,33 @@ export class HdIdentityVault implements IdentityVault<{ InitializeResult: string
     // created it will use the derived keys.
     const deterministicKeyGenerator = new DeterministicKeyGenerator();
     await deterministicKeyGenerator.addPredefinedKeys({
-      privateKeys: [ identityPrivateKey, signingPrivateKey]
+      privateKeys: [identityPrivateKey, signingPrivateKey]
     });
 
-    // Create the DID using the derived identity, signing, and encryption keys.
-    const did = await DidDht.create({
-      keyManager : deterministicKeyGenerator,
-      options    : {
-        verificationMethods: [
-          {
-            algorithm : 'Ed25519',
-            id        : 'sig',
-            purposes  : ['assertionMethod', 'authentication']
-          },
-          // TODO: Enable this once DID DHT supports X25519 keys.
-          // {
-          //   algorithm : 'X25519',
-          //   id        : 'enc',
-          //   purposes  : ['keyAgreement']
-          // }
-        ]
+  // Create the DID using the derived identity, signing, and encryption keys.
+  const options = {
+    verificationMethods: [
+      {
+        algorithm : 'Ed25519',
+        id        : 'sig',
+        purposes  : ['assertionMethod', 'authentication']
+      },
+    ]
```

Comment:
@frankhinek Any objections to adding `secp256k1` keys as encryption keys here? That's the curve that is supported in `dwn-sdk-js`.

```suggestion
    verificationMethods: [
      {
        algorithm : 'Ed25519',
        id        : 'sig',
        purposes  : ['assertionMethod', 'authentication']
      },
      {
        algorithm : 'secp256k1',
        id        : 'enc',
        purposes  : ['keyAgreement']
      } 
    ]
```